crates/iota/src/iota_commands.rs

```diff
@@ -1192,42 +1192,32 @@ fn prompt_for_environment(
             print!(
                 "Creating config file [{wallet_conf_path:?}] with default (Testnet) full node server and ed25519 key scheme."
             );
-        } else {
-            print!(
-                "Config file [{wallet_conf_path:?}] doesn't exist! Do you want to connect to an IOTA full node server? [y/N]: "
-            );
+            return Ok(IotaEnv::testnet());
         }
-        if accept_defaults || matches!(read_line(), Ok(line) if line.trim().to_lowercase() == "y") {
-            let url = if accept_defaults {
-                String::new()
-            } else {
-                print!(
-                    "Select a default network [mainnet|testnet|devnet|localnet], or enter a custom IOTA full node server URL (defaults to testnet if not specified): "
-                );
-                read_line()?
-            };
-            match url.trim() {
-                "mainnet" => Ok(IotaEnv::mainnet()),
-                "testnet" | "" => Ok(IotaEnv::testnet()),
-                "devnet" => Ok(IotaEnv::devnet()),
-                "localnet" => Ok(IotaEnv::localnet()),
-                url => {
-                    if Url::parse(url).is_ok() {
-                        print!("Environment alias for [{url}]: ");
-                        let alias = read_line()?;
-                        let alias = if alias.trim().is_empty() {
-                            "custom".to_string()
-                        } else {
-                            alias
-                        };
-                        Ok(IotaEnv::new(alias, url))
+
+        print!(
+            "Select a default network [mainnet|testnet|devnet|localnet], or enter a custom IOTA full node server URL (defaults to testnet if not specified): "
+        );
+        let url = read_line()?;
+        match url.trim() {
```

Comment:
I didnt think of it in my PR but maybe also lowercase the input here.

Reply:
Sure